ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+### Context
+
+* [ ] **mapbox-gl-js version**: 
```

Comment:
But if they use an out-of-date version, what's the point of reporting an issue in the first place? If an issue is already fixed on the master version, then a report will be immediately closed, and if it's not fixed, it would be nice to ask the user to make sure instead of checking it ourselves.